stringutil/strings.go

```diff
@@ -51,3 +52,32 @@ func GetFilterType(v interface{}) (reflect.Type, error) {
 	}
 	return nil, errors.New("Filter must be struct or slice of structs")
 }
+
+// Table alias counter used to increment table aliases as we
+// progress through a query / join, etc.
+var (
+	tableAliasIndex = 0
```

Comment:
Using a global as the counter like this is not thread-safe. If warden gets multiple query requests at the same time, or if queries are built inside goroutines, the aliases are going to be all over the place and incorrect. The tests pass because they run one at a time.

I'd suggest you pass `0` as the first index value to the first call to `Build()` instead, and make sure all the subsequent passes and increments happen by pointer reference, not by value.